coins.json

```diff
@@ -705,6 +705,23 @@
             "clientDocs": "https://www.stellar.org/developers/horizon/reference"
         }
     },
+    {
+        "id": "nuls",
+        "name": "NULS",
+        "symbol": "NULS",
+        "decimals": 8,
+        "blockchain": "NULS",
+        "derivationPath": "m/44'/8964'/0'/0/0",
+        "curve": "secp256k1",
+        "publicKeyType": "secp256k1",
+        "explorer": "https://nulscan.io/",
+        "info": {
+            "url": "https://nuls.io",
+            "client": "https://github.com/nuls-io/nuls-v2",
+            "clientPublic": "",
```

Comment:
could you please provide this https://developer.trustwallet.com/wallet-core/rpc-requirements?

Reply:
OK, nuls 2.0 has public service api link https://public1.nuls.io/